pkg/tfbridge/diff_test.go

```diff
@@ -1903,5 +2008,80 @@ func TestListNestedAddMaxItemsOne(t *testing.T) {
 		// expected
 		map[string]DiffKind{
 			"prop.nest": AR,
-		})
+		},
+		pulumirpc.DiffResponse_DIFF_SOME)
+}
+
+func TestChangingTagsAll(t *testing.T) {
```

Comment:
Does this fail when you remove the new changes?

Reply:
It fails when I remove the changes _except_ what Anton raised above.